script/ProtocolRewards.s.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.19;
+
+import {Script} from "forge-std/Script.sol";
+import {ProtocolRewards} from "../contracts/ProtocolRewards.sol";
+import {ERC1967FactoryConstants} from "solady/src/utils/ERC1967FactoryConstants.sol";
+import {ERC1967Factory} from "solady/src/utils/ERC1967Factory.sol";
+
+// # To deploy and verify ProtocolRewards.sol run this command below
+// forge script script/ProtocolRewards.s.sol:ProtocolRewardsDeploy --rpc-url sepolia --broadcast --verify -vvvv
+contract ProtocolRewardsDeploy is Script {
+    function run() external {
+        uint256 deployerPrivateKey = vm.envUint("MAINNET_PRIVATE_KEY");
```

Comment:
At some point it might be interesting to look into Frame as deploy/upgrade signer vs importing it into the environment. Out of scope here for sure though.